scripts/html5.js

```diff
@@ -489,8 +502,21 @@ H5P.VideoHtml5 = (function ($) {
       if (lastState === undefined) {
         // Make sure we always play before we seek to get an image.
         // If not iOS devices will reset currentTime when pressing play.
-        video.play();
-        video.pause();
+
+        // Always play to get image.
+        var playPromise = video.play();
+        if (playPromise !== undefined) {
+          playPromise
+            .then(function () {
+              video.pause();
+            })
+            .catch(function (error) {
+              video.pause(); // Fallback
+            });
```

Comment:
Same as the above comment:
playPromise.finally(() => video.pause());

Reply:
@fnoks Makes total sense! Done.